easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -240,12 +242,18 @@ def __init__(self, *args, **kwargs):
             self.use_setup_py = True
 
             if self.cfg.get('use_setup_py_develop', False):
+                self.log.deprecated("Use 'install_target' rather than 'use_setup_py_develop'.", '4.0')
                 self.install_cmd = SETUP_PY_DEVELOP_CMD
             else:
                 self.install_cmd = SETUP_PY_INSTALL_CMD
 
+            if self.cfg['install_target'] == 'easy_install':
```

Comment:
introduce a `EASY_INSTALL_TARGET` constant on top, and use that (also use it to define `EASY_INSTALL_INSTALL_CMD`)